fdbclient/vexillographer/fdb.options

```diff
@@ -210,6 +213,8 @@ description is not currently required but encouraged.
             description="Sets a client provided identifier for the transaction that will be used in scenarios like tracing or profiling. Client trace logging or transaction profiling must be separately enabled." />
     <Option name="log_transaction" code="404"
             description="Enables tracing for this transaction and logs results to the client trace logs. The DEBUG_TRANSACTION_IDENTIFIER option must be set before using this option, and client trace logging must be enabled and to get log output." />
+    <Option name="transaction_logging_max_field_length" code="405" paramType="Int" paramDescription="Maximum length of escaped key and value fields."
```

Comment:
I think for most of these options, the name omits "transaction", and then it is only the database option that adds it back. (So `DatabaseOptions::transaction_time_limit` affects `TransactionOptions::time_limit`.) So this would be `logging_max_field_length`, I think.

Reply:
Yeah, this is a bit tricky because the `transaction_` prefix isn't intended to describe the type of option but rather the type of logging. In particular, this doesn't affect field lengths for trace logs in general but only for fields in the transaction logging enabled by `log_transaction`.

Maybe we could reorder the words, like `max_transaction_logging_field_length`? Other suggestions welcome, too.

Reply:
Hm, maybe it's fine as is, then. I don't think putting max at the beginning helps, and maybe it is useful to point out this is for transaction logging in particular.